drafts/builtins-sql.adoc

```diff
@@ -0,0 +1,424 @@
+= SQL Builtins
+:table-caption!:
+
+== Preface
+This document contains the standard functions as specified by the SQL standard. For spec conformance, a PartiQL implementation must contain all of these. This document contains an extended function section which includes additional functions which are not in the SQL spec, but are common amongst SQL implementations.
```

Comment:
Since we have a imperative voice in this paragraph and it's not an RFC yet, does it make sense to change the title to something like `[DRAFT] PartiQL supported SQL Builtins`?

Reply:
I meant this doc to be only for reference rather than our specification. I could see this being a useful spec appendix however. I can drop the imperative

> For spec conformance, a PartiQL implementation must contain all of these. 

Then this doc is purely SQL reference material agnostic to PartiQL